openfl-workspace/torch_cnn_mnist/plan/plan.yaml

```diff
@@ -1,45 +1,69 @@
-# Copyright (C) 2020-2021 Intel Corporation
+# Copyright (C) 2020-2024 Intel Corporation
 # Licensed subject to the terms of the separately executed evaluation license agreement between Intel Corporation and you.
 
-aggregator :
-  defaults : plan/defaults/aggregator.yaml
-  template : openfl.component.Aggregator
-  settings :
-    init_state_path     : save/torch_cnn_mnist_init.pbuf
-    best_state_path     : save/torch_cnn_mnist_best.pbuf
-    last_state_path     : save/torch_cnn_mnist_last.pbuf
-    rounds_to_train     : 10
-    log_metric_callback :
-      template : src.utils.write_metric
-
-
-collaborator :
-  defaults : plan/defaults/collaborator.yaml
-  template : openfl.component.Collaborator
-  settings :
-    delta_updates    : false
-    opt_treatment    : RESET
-
-data_loader :
-  defaults : plan/defaults/data_loader.yaml
-  template : src.dataloader.PyTorchMNISTInMemory
-  settings :
-    collaborator_count : 2
-    data_group_name    : mnist
-    batch_size         : 256
-
-task_runner :
-  defaults : plan/defaults/task_runner.yaml
-  template : src.taskrunner.PyTorchCNN
-
-network :
-  defaults : plan/defaults/network.yaml
-
-assigner :
-  defaults : plan/defaults/assigner.yaml
-  
-tasks :
-  defaults : plan/defaults/tasks_torch.yaml
-
-compression_pipeline :
-  defaults : plan/defaults/compression_pipeline.yaml
+aggregator:
+  settings:
+    best_state_path: save/best.pbuf
+    db_store_rounds: 2
+    init_state_path: save/init.pbuf
+    last_state_path: save/last.pbuf
+    rounds_to_train: 2
+    write_logs: false
+  template: openfl.component.aggregator.Aggregator
+assigner:
+  settings:
+    task_groups:
+    - name: train_and_validate
+      percentage: 1.0
+      tasks:
+      - aggregated_model_validation
+      - train
+      - locally_tuned_model_validation
+  template: openfl.component.RandomGroupedAssigner
+collaborator:
+  settings:
+    db_store_rounds: 1
+    delta_updates: false
+    opt_treatment: RESET
+  template: openfl.component.collaborator.Collaborator
+compression_pipeline:
+  settings: {}
+  template: openfl.pipelines.NoCompressionPipeline
+data_loader:
+  settings:
+    batch_size: 64
+    collaborator_count: 2
+  template: src.dataloader.PyTorchMNISTInMemory
+network:
+  settings:
+    agg_addr: localhost
+    agg_port: 59583
+    cert_folder: cert
+    client_reconnect_interval: 5
+    disable_client_auth: false
```

Comment:
As per [`openfl/openfl-workspace/workspace/plan/defaults/network.yaml`](https://github.com/securefederatedai/openfl/blob/develop/openfl-workspace/workspace/plan/defaults/network.yaml#L6), the variables are `use_tls` and `require_client_auth` instead of `tls` and `disable_client_auth` respectively.

Reply:
resolved in af74b48cfa74c757c0002555c8022d12156173ce